bin/phpbench.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```

Comment:
Kind of. I can do this:
```
C:\Work\phpbench>bin\phpbench
phpbench version 0.10.0-dev
```

But when starting a process using `new Process` it doesn't work. Removing that bat file changes this:
`Tests: 245, Assertions: 461, Errors: 17, Failures: 31.`
to this:
`Tests: 245, Assertions: 504, Errors: 5, Failures: 15.`